lib/etso/ets/match_specification.ex

```diff
@@ -59,6 +59,15 @@ defmodule Etso.ETS.MatchSpecification do
     {:==, build_condition(field_names, params, field), nil}
   end
 
+  defp build_condition(field_names, _, {:json_extract_path, [], paths}) do
```

Comment:
This can possibly be simplified/prettified, if we know the calling convention of `:json_extract_path`

Reply:
this is what the query is translated to.